fla/ops/path_attn/parallel_path_bwd_inter_dkv.py

```
@@ -75,97 +75,56 @@ def parallel_path_bwd_dkv_kernel(
     b_dv = tl.zeros([BT, K], dtype=tl.float32)
     idx_i = (i_t * BT // S).to(tl.int32)
```

Ruff (0.11.9)

Local variable `idx_i` is assigned to but never used

Remove assignment to unused variable `idx_i`

(F841)